kubernetes/annotations_and_labels.md

```diff
@@ -6,6 +6,9 @@
   location. For now this should be in our `giantswarm/giantswarm` repository in
   which we crosslink all pages to create a reasonable documentation of all kinds
   of Kubernetes resources, their functionality and relationships.
+- `giantswarm.io/version-bundle` - for CRs exposed to customers we will use
```

Comment:
Let's please replace it with just `version`. It will make more sense with versioning of operators and in general. This is just a release version or operator version. `bundle` is just a legacy word. I also would express what it's actually is and it is going to replace `.spec.versionBundle` completely.

```suggestion
- `giantswarm.io/version` - for CRs exposed to customers we will use
```

Reply:
Yes I see this as a stopgap solution until we move to `apiVersion`.

It keeps the new CRDs cleaner by moving the version to an annotation. Using `version-bundle` as the name matches the terminology we use currently.